src/Raven.Client/Documents/Indexes/IAttachmentObject.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text;
+
+namespace Raven.Client.Documents.Indexes
+{
+    public interface IAttachmentObject
+    {
+        public string Name { get; }
+
+        public string Hash { get; }
+
+        public string ContentType { get; }
+
+        public long Size { get; }
+
+        public string GetContentAsString();
+
+        public string GetContentAsString(Encoding encoding);
```

Comment:
Should we expose a `Stream` here as well?

Reply:
Yes, wrote that in the comment above.